src/js/jsx/shared/Dialog.jsx

```diff
@@ -101,12 +101,13 @@ define(function (require, exports, module) {
                 openDialogs = dialogState.openDialogs;
 
             return {
-                open: openDialogs.has(this.props.id)
+                open: openDialogs.has(this.props.id + this._uniqkey)
             };
         },
 
         componentWillMount: function () {
-            this.getFlux().store("dialog").registerDialog(this.props.id, this._getDismissalPolicy());
+            this._uniqkey = (new Date()).getTime();
```

Comment:
Can you declare `_uniqKey` at the top of component with JSDoc, and reasons for it's existence?